### PR TITLE
Spanish Translation for HOWTOPLAY and README

### PR DESCRIPTION
**Pull request by Julielinas-github**
what was done:

I added spanish translations for the HOW_TO_PLAY and README files and added .svg buttons for each on the respective pages

### CODE REVIEW

File: HOW_TO_PLAY.md [hunk 1]
```diff
@@ -152,4 +153,4 @@ If you're able to obtain the wood and leaves, it also makes the perfect deterant
 
 They're bouncy and delicious, and pigs absolutely love them!
 
-<!-- https://media.tenor.com/GTYRNMAuLIUAAAAC/muffin-button-muffins.gif -->
\ No newline at end of file
+<!-- https://media.tenor.com/GTYRNMAuLIUAAAAC/muffin-button-muffins.gif -->
```
Comment (Sollace) [inline HOW_TO_PLAY.md:156]: 👀

**Comment by Sollace** [comment]
Linking #439

**Review by MimzyFang — COMMENTED** [review]
I think this is the wrong branch

**Comment by Sollace** [comment]
Please don't push unrelated changes.

Since this was already trying to sneak things in with that hidden image, I now have to doubt the rest of the changes. Since it's translations I cannot verify.

**Comment by Julielinas-github** [comment]
Sorry about that, I didn't mean to post that enhancement file publicly. I was trying out building my own version of the mod and I didn't realize it would add to this pull request. I completely understand not trusting what looks like sneaking in malicious code or bad translation, I just wanted to say sorry about giving you a jumpscare with a 2AM update.

**Pull request closed without merging**